src/Umbraco.Web/Editors/MediaController.cs

```diff
@@ -278,6 +281,31 @@ public IEnumerable<MediaItemDisplay> GetByIds([FromUri]int[] ids)
         }
 
         /// <summary>
+        /// Searches media and returns a paged result
+        /// </summary>
+        /// <param name="query"></param>
+        /// <param name="pageNumber"></param>
+        /// <param name="pageSize"></param>
+        /// <param name="searchFrom"></param>
+        /// <returns></returns>
+        [HttpGet]
+        [FilterAllowedOutgoingMedia(typeof(IEnumerable<ContentItemBasic<ContentPropertyBasic, IMedia>>), "Items")]

```

Comment:
@madsrasmussen is there a reason we can't 'just' use the normal search: EntityController.Search or modify it to suit our needs for paging? Doing this has security consequences along with a ton of code duplication. This would mean that if a user doesn't have access to the media section they will not be able to search it. This is a rare case but it can happen. The EntityController is readonly and because it only returns a subset of data isn't locked down to user's with access to certain areas.